Salesforce/client/token_refresher.py

```diff
@@ -112,11 +113,18 @@ async def refresh_token(self) -> None:
         async with self.session().post(url, auth=BasicAuth(self.client_id, self.client_secret), json={}) as response:
             response_data = await response.json()
 
-            self._token = SalesforceToken(
-                token=HttpToken(**response_data),
-                created_at=time.time(),
-                ttl=self.token_ttl,
-            )
+            try:
+                self._token = SalesforceToken(
+                    token=HttpToken(**response_data),
+                    created_at=time.time(),
+                    ttl=self.token_ttl,
+                )
+            except Exception as e:
+                logger.info(
+                    "Cannot get token. Response contains {0} status {1}".format(response.status, response_data)
```

Comment:
I think you wanted to display the content before the status
```suggestion
                    "Cannot get token. Response contains {0} status {1}".format(response_data, response.status)
```